app/models/job.ts

```diff
@@ -600,17 +634,24 @@ export class Job extends Record implements JobRecord {
   }
 
   /**
-   * Updates the status to paused providing the optional message or the default
-   * if none is provided.  You should generally provide a message if possible.
+   * Updates the status to paused.
    * Only jobs in the RUNNING state may be paused.
    * You must call `#save` to persist the change.
    *
-   * @param message - a human-readable message to indicate the state of the job
    * @throws An error if the job is not currently in the RUNNING state
    */
-  pause(message = statesToDefaultMessages.paused): void {
+  pause(): void {
     validateTransition(this.status, JobStatus.PAUSED, JobEvent.PAUSE);
-    this.updateStatus(JobStatus.PAUSED, message);
+    let newMessage = `${statesToDefaultMessages[JobStatus.PAUSED]}.`;
+    const messagePartsToRemove = activeJobStatuses.map((status) => statesToDefaultMessages[status]);
+    const retainedMessage = removeMessageParts(
+      this.message,
+      messagePartsToRemove,
+      (part) => part.includes('CMR query identified'));
```

Comment:
Oh we're using includes... so it is pattern matching and then the splitting on "." may be problematic if we change messages in the future.